autobahn/asyncio/component.py

```diff
@@ -49,19 +48,11 @@
 from autobahn.wamp import component
 
 from autobahn.asyncio.wamp import Session
-from autobahn.wamp.exception import ApplicationError
 
 
 __all__ = ('Component',)
 
 
-def _is_ssl_error(e):
-    """
-    Internal helper.
-    """
-    return isinstance(e, ssl.SSLError)
-
-
 def _unique_list(seq):
```

Comment:
Is there a reason this was moved to be an instance-method instead of a bare helper function?

Reply:
So the tx/aio versions of _is_ssl_error() can be called by the component base class _start() method.

Reply:
Ah, okay thanks that makes sense